packages/nx/src/migrations/update-17-0-0/rm-default-collection-npm-scope.ts

```diff
@@ -0,0 +1,23 @@
+import { formatChangedFilesWithPrettierIfAvailable } from '../../generators/internal-utils/format-changed-files-with-prettier-if-available';
+import { Tree } from '../../generators/tree';
+import { readNxJson, updateNxJson } from '../../generators/utils/nx-json';
+
+export default async function update(tree: Tree) {
+  if (!tree.exists('nx.json')) {
+    return;
+  }
+
+  const nxJson = readNxJson(tree);
+
+  delete nxJson.cli?.['defaultCollection'];
+
+  if (Object.keys(nxJson.cli).length < 1) {
+    delete nxJson.cli;
+  }
+
+  delete nxJson['npmScope'];
+
+  updateNxJson(tree, nxJson);
```

Comment:
Can we update the root package.json name so that the npm scope inference works properly if its **_not_** an Nx project? If it was a project that would be dangerous to do, but should be fine it its not right?

Reply:
Huh...